decentraland/unity-renderer
# feat: added report bug button

## What does this PR change?

Fix #1178 

## How to test the changes?

Use this url
https://play.decentraland.zone/?renderer-branch=feat/report-bug&kernel-branch=feat/renderer-config-versions

- Open taskbar menu
- Click on Report Bug
- Form should be filled with unity version, kernel version, nametag, system specs

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
